src/android/shakeDetector.java

```diff
@@ -106,43 +110,27 @@ public void onSensorChanged(SensorEvent event){
                 acceleration = acceleration * 0.95f + (currAcceleration - lastAcceleration);
                 String AccTag1 = "2Shake0.6A0.95-13-10";
 
-                Log.i(this, AccTag1, ","+String.valueOf(currTime)+"," + Float.toString(xAcceleration) + "," + Float.toString(yAcceleration) + "," + Float.toString(zAcceleration) + "," + Float.toString(acceleration) + ",");
+                //Log.i(this, AccTag1, ","+String.valueOf(currTime)+"," + Float.toString(xAcceleration) + "," + Float.toString(yAcceleration) + "," + Float.toString(zAcceleration) + "," + Float.toString(acceleration) + ",");
 
-                /*
                 if ((acceleration > 13f) && (Math.abs(xAcceleration) < 10)) {
                     lastUpdated = currTime;
-                    NotificationHelper.createNotification(this, Constants.TRACKING_ERROR_ID, AccTag1);
-                    //Log.i(this, AccTag1, ","+String.valueOf(currTime)+"," + Float.toString(xAcceleration) + "," + Float.toString(yAcceleration) + "," + Float.toString(zAcceleration) + "," + Float.toString(acceleration) + ",");
-                    notifyEvent(this, AccTag1, new JSONObject());
+                    //NotificationHelper.createNotification(this, Constants.TRACKING_ERROR_ID, AccTag1);
+                    Log.i(this, AccTag1, ","+String.valueOf(currTime)+"," + Float.toString(xAcceleration) + "," + Float.toString(yAcceleration) + "," + Float.toString(zAcceleration) + "," + Float.toString(acceleration) + ",");
+                    notifyEvent(this, POTENTIAL_INCIDENT, new JSONObject());
```

Comment:
future fix: you could potentially have the `'data'` part of the event represent the sensor that detected the gesture - e.g. {'sensor': 'accelerometer'}